metron-platform/metron-elasticsearch/src/main/java/org/apache/metron/elasticsearch/utils/ElasticsearchUtils.java

```diff
@@ -179,4 +186,46 @@ else if(ipObj instanceof List) {
     }
     throw new IllegalStateException("Unable to read the elasticsearch ips, expected es.ip to be either a list of strings, a string hostname or a host:port string");
   }
+
+  /**
+   * Converts an Elasticsearch SearchRequest to JSON.
+   * @param esRequest The search request.
+   * @return The JSON representation of the SearchRequest.
+   */
+  public static String toJSON(org.elasticsearch.action.search.SearchRequest esRequest) {
+    String json = "null";
```

Comment:
Should we instead return back an `Optional<String>` rather than the string representation of null on a failure here?  Alternatively, maybe throw an exception so the calling function can handle the exception how it likes?

Reply:
Yes, definitely this is kind of weird.  This is taken care of in the latest commit.